RunForrest/src/css/_colors.scss

```diff
@@ -0,0 +1,11 @@
+$grey: #8e8e8e;;
+$dark-grey: #666666;
+$grey34: #575757;
```

Comment:
Try using more significant names. Like lighter, darker, etc. Or choose a 'color name' from http://chir.ag/projects/name-that-color/#575757 to name it `grey-scorpion` for example.